apps/fares/lib/fare_info.ex

```diff
@@ -5,28 +5,62 @@ defmodule Fares.FareInfo do
 
   alias Fares.Fare
 
+  now = DateTime.utc_now()
+  surge_start = ~U[2022-08-20 01:00:00Z]
+  surge_end = ~U[2022-09-19 06:00:00Z]
+
   @fare_data [
     %{
       mode: :commuter,
       zone: "1A",
-      single_trip: "2.40",
-      single_trip_reduced: "1.10",
+      single_trip:
+        if DateTime.compare(now, surge_start) == :gt and DateTime.compare(now, surge_end) == :lt do
```

Comment:
suggestion(non-blocking): Since this logic is used multiple times if you wanted to you could just call it once, up in line 11 or so, and assign it to a variable, say `in_surge_window?` or something. Then each of these checks could just be `if in_surge_window? do`.